bugbot/rules/not_landed.py

```diff
@@ -349,7 +329,7 @@ def get_bugs(self, date="today", bug_ids=[]):
             nickname = d["nickname"]
 
             if not assignee:
-                assignee, nickname = nicknames[bugid]
+                assignee, nickname = bz_reviewers[nicknames[bugid]]
```

Comment:
Splitting this in two lines would improve the readability:
```suggestion
                assignee_id = nicknames[bugid]
                assignee, nickname = bz_reviewers[assignee_id]
```

Reply:
split for enhanced readability